### PR TITLE
start docs with sphinx

### PR DESCRIPTION
**Pull request by evgeni**
preview at https://apypie.readthedocs.io/en/docs/

### CODE REVIEW

**Comment by evgeni** [comment]
@sean797 would love your thoughts :)

File: apypie/api.py [hunk 1]
```diff
@@ -14,6 +14,20 @@
 class Api:
     """
     Apipie API bindings
+
+    :param uri: base URL of the server
+    :param username: username to access the API
+    :param password: username to access the API
+    :param api_version: version of the API. Defaults to `1`
```
[the remark this answers is not in the document]
Comment (evgeni) [reply apypie/api.py]: absolutely!

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
```
Comment (sean797) [inline docs/conf.py]: I guess I forgot to remove these comments

Comment (evgeni) [reply docs/conf.py]: You? Nope, but I did ;)

**Comment by evgeni** [comment]
This is already automatically triggered, yes.

**Pull request merged**